templates/EntityAcl/idps.html.twig

```diff
@@ -15,7 +15,9 @@
             {% endfor %}
         {% endfor %}
 
-        {{ form_errors(form) }}
+        {% for error in form.vars.errors %}
```

Comment:
Is this custom code construct really needed?

Reply:
@teunfransen @quartje 
Should this validation also be added this screen?
![image](https://github.com/user-attachments/assets/dea078d9-aeaa-4d37-a44d-b554cf93d256)

@quartje 
Teun mentioned that unchecking `Select all` is similar to not selecting anything (not sure if I understood that correctly). So it might not cause functional issues? But it might not be good UX.

Reply:
I've tested this, and it's true